wsidicom/series/pyramids.py

```diff
@@ -109,6 +110,36 @@ def _group_instances_into_pyramids(
                 )
             else:
                 ext_depth_of_field = None
+
+            def _cs_matches(cs_a, cs_b):
+                return (
```

Comment:
I would make the calculation more explicit by assigning the difference to a variable before comparision, and also include the square root.